config/storybook-preset-carbon/carbon-theme-addon/src/preset.js

```diff
@@ -0,0 +1,16 @@
+/**
+<<<<<<< HEAD
```

Comment:
Looks like a few merge artifacts are here 🕵️

Reply:
Omg i fixed those. 🙃 thanks for catching